src/unistd.rs

```diff
@@ -1144,6 +1144,40 @@ pub fn unlink<P: ?Sized + NixPath>(path: &P) -> Result<()> {
     Errno::result(res).map(drop)
 }
 
+/// Flags for `unlinkat` function.
+#[derive(Clone, Copy, Debug)]
+pub enum UnlinkatFlags {
+    RemoveDir,
+    NoRemoveDir,
+}
+
+/// Remove a directory entry
+///
+/// In the case of a relative path, the directory entry to be removed is determined relative to
+/// the directory associated with the file descriptor dirfd. If flag is UnlinkatFlags::RemoveDir
```

Comment:
Symbols like `dirfd`, `flag`, and `UnlinkatFlags::RemoveDir` should be in backticks.  The doc comment should also explain what it means for `dirfd` to be `None`.